data/feedbackRepository.go

```diff
@@ -1,23 +1,59 @@
 package data
 
 import (
-	"database/sql"
 	"github.com/UHERO/rest-api/models"
+	"log"
+	"os"
+	"fmt"
+	"github.com/andygrunwald/go-jira"
 )
 
 type FeedbackRepository struct {
-	DB *sql.DB
 }
 
-func (r *FeedbackRepository) CreateFeedback(feedback models.Feedback) (numRows int64, err error) {
-	stmt, err := r.DB.Prepare(`INSERT INTO user_feedbacks(name, email, feedback, created_at, updated_at)
-	VALUES (?, ?, ?, NOW(), NOW());`)
+func (r *FeedbackRepository) CreateFeedback(feedback models.Feedback) (err error) {
+	jiraClient, err := jira.NewClient(nil, "https://uhero-analytics.atlassian.net")
 	if err != nil {
+		log.Printf("JIRA connection error: %s", err)
 		return
 	}
-	res, err := stmt.Exec(feedback.Name, feedback.Email, feedback.Feedback)
+
+	res, err := jiraClient.Authentication.AcquireSessionCookie(os.Getenv("JIRA_USER"), os.Getenv("JIRA_PASSWORD"))
+	if err != nil || res == false {
+		fmt.Printf("JIRA Result: %v\n", res)
```

Comment:
Do you want this (and below) log-outs to go to stdout (which is where I assume fmt sends them), or to the log, as in line 17?